tests/Feature/MeetingTest.php

```diff
@@ -0,0 +1,209 @@
+            ]
+        );
+
+        $response
+            ->assertStatus(201)
+            ->assertJson([
+                'success' => true
+            ]);
+    }
+
+    public function test_index()
+    {
+        $organizer = factory(User::class)->create();
+        $user = factory(User::class)->create();
+
+        $meeting = factory(Meeting::class)->make();
+        $organizer->meetings()->save($meeting);
+
+        $response = $this->actingAs($user, 'api')->getJson(
+            route('api.meeting.list')
+        );
+
+        $response
+            ->assertStatus(200)
+            ->assertJson([
+                'success' => true,
+                'rowCount' => 1
+            ]);
+    }
+
+    public function test_search_found()
+    {
+        $organizer = factory(User::class)->create();
+        $user = factory(User::class)->create();
+
+        $meeting1 = factory(Meeting::class)->make();
+        $organizer->meetings()->save($meeting1);
+        $meeting2 = factory(Meeting::class)->make(['name' => 'My Fake Meetings']);
+        $organizer->meetings()->save($meeting2);
+
+        $response = $this->actingAs($user, 'api')->getJson(
+            route(
+                'api.meeting.list',
+                [
+                    'name' => $meeting1->name
+                ]
+            )
+        );
+
+        // search for partial name
+        $response
+            ->assertStatus(200)
+            ->assertJson([
+                'success' => true,
+                'rowCount' => 1
+            ]);
+
+        $response = $this->actingAs($user, 'api')->getJson(
+            route('api.meeting.list',
+            [
+                'name' => 'Fake Meeting'
+            ])
+        );
+
+        $response
+            ->assertStatus(200)
+            ->assertJson([
+                'success' => true,
+                'rowCount' => 1
+            ]);
+    }
+
+    public function test_search_notFound()
+    {
+        $organizer = factory(User::class)->create();
+        $user = factory(User::class)->create();
+
+        $meeting = factory(Meeting::class)->make();
+        $organizer->meetings()->save($meeting);
+
+        $response = $this->actingAs($user, 'api')->getJson(
+            route(
+                'api.meeting.list',
+                [
+                    'name' => 'Wrong Meeting Name'
+                ]
+            )
+        );
+
+        $response
+            ->assertStatus(200)
+            ->assertJson([
+                'success' => true,
+                'rowCount' => 0
+            ]);
+    }
+
+    public function test_update()
+    {
+        $organizer = factory(User::class)->create();
+
+        $meeting = factory(Meeting::class)->make();
+        $organizer->meetings()->save($meeting);
+
+        $response = $this->actingAs($organizer, 'api')->putJson(
+            route('api.meeting.update', ['meeting' => $meeting->getKey()]),
+            [
+                'name' => 'Fake Meeting'
+            ]
+        );
```

Comment:
Can we assert the meeting got updated? `$meeting = Meeting::first()` then `$this->assertEquals('Fake Meeting', $meeting->name)` etc.